readthedocs/templates/builds/build_detail.html

```diff
@@ -79,10 +79,17 @@
 
     <div class="build-version">
       <span class="build-version-slug">
-        {% if request.user|is_admin:project %}
+        {% if request.user|is_admin:project and not build.is_external %}
           <a href="{% url "project_version_detail" build.version.project.slug build.version.slug %}">{{ build.version.slug }}</a>
         {% else  %}
-          {{ build.version.slug }}
+          {% if build.is_external %}
```

Comment:
shouldn't this just be an `{% else if %}` on the top-level `if`? This is more complex to read.